### ISSUE TITLE
Golden Walnut and Qi Gems Count

### ISSUE THREAD

**Issue by squignon** [stardew-access/stardew-access#526]
On the Skills menu there is an icon and number for the golden walnuts as well as Qi Gems but since you cannot navigate the cursor to it there does not seem to be a way to get a screen reader to read the counts out loud. Would it be possible to have them read out automatically as soon as the skills menu is opened?

**Comment by khanshoaib3** [comment]
Have you tried using the `C` key when in skills page? That should speak up the counts of Golden walnut, Qi related stuff and more.

I'm closing this issue, thanks.
